tsconfig.json

```diff
@@ -3,7 +3,6 @@
     "target": "ES6",
     "module": "CommonJS",
     "moduleResolution": "Node",
-    "esModuleInterop": true,
```

Comment:
using this forces it on consumers as well

Reply:
Seems a good idea 😊